lang/en/tool_mfa.php

```diff
@@ -43,10 +43,13 @@
 $string['error:revoke'] = 'Can not revoke factor';
 $string['error:notenoughfactors'] = 'Not enough factors to authenticate';
 $string['error:factornotenabled'] = 'MFA Factor \'{$a}\' not enabled';
+$string['error:notenoughsetup'] = 'Not enough factors setup to login';
```

Comment:
looks like duplicate of 

$string['error:notenoughfactors'] = 'Not enough factors to authenticate';